### PR TITLE
added TransactionsPage tests

### PR DESCRIPTION
**Pull request by Errba**

### CODE REVIEW

File: src/test/java/pages/TransactionsPage.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
```
Comment (beckykalambek) [inline src/test/java/pages/TransactionsPage.java:3]: perfect)

File: src/test/java/tests/TransactionsTests.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package tests;
+
+import org.testng.annotations.Test;
+
+public class TransactionsTests {
+
+
+
+    @Test
+    public void transactionsTest1 () {
```
Comment (Annapurna80) [inline src/test/java/tests/TransactionsTests.java:10]: You are doing great!

File: src/test/java/tests/TransactionsTests.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package tests;
+
+import org.testng.annotations.Test;
+
+public class TransactionsTests {
+
+
```
Comment (beckykalambek) [inline src/test/java/tests/TransactionsTests.java:7]: hgghhghgh

File: src/test/java/pages/TransactionsPage.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package pages;
+
```
Comment (marinawittich) [inline src/test/java/pages/TransactionsPage.java:2]: Looks good

**Review by marinawittich — APPROVED** [review]

**Review by beckykalambek — APPROVED** [review]

**Review by marinawittich — APPROVED** [review]

**Pull request merged**